catalog/pom.xml

```diff
@@ -49,7 +49,7 @@
         <catalog.joda.time.version>2.2</catalog.joda.time.version>
         <lucene.bundle.version>5.1.0_1</lucene.bundle.version>
         <lucene.version>5.1.0</lucene.version>
-        <mina.version>2.0.13</mina.version>
+        <mina.version>2.0.6</mina.version>
```

Comment:
@stustison Searched "Mina" and "Apache Mina" on the CVE site and no relevant results were returned for it.